packages/block-library/src/quote/transforms.js

```diff
@@ -74,44 +153,47 @@ const transforms = {
 				return (
 					node.nodeName === 'BLOCKQUOTE' &&
 					// The quote block can only handle multiline paragraph
```

Comment:
Yeah, the code here has changed, but the comment was missed. I think the comment can be removed entirely since the code is pretty self-explanatory now anyway.